generators/app/templates/_.npmignore

```diff
@@ -0,0 +1 @@
+!dist/
```

Comment:
I don't think it means that. Otherwise, the advice for overriding `.gitignore` wouldn't be to create an empty `.npmignore`.

Also, `npm install -g .` on a generated plugin includes everything (with `dist/`).